components/dashboard/src/projects/Project.tsx

```diff
@@ -356,6 +357,22 @@ export default function () {
                                                     <div className="text-base text-gray-500 dark:text-gray-50 font-medium mb-1 truncate">
                                                         {shortCommitMessage(branch.changeTitle)}
                                                     </div>
+                                                    {branch.changeDate ? (
+                                                        <Tooltip
+                                                            className="w-fit"
+                                                            content={dayjs(branch.changeDate).format("MMM D, YYYY")}
+                                                        >
+                                                            <p>
+                                                                {avatar}Authored {formatDate(branch.changeDate)} ·{" "}
+                                                                {branch.changeHash?.substring(0, 8)}
+                                                            </p>
+                                                        </Tooltip>
+                                                    ) : (
+                                                        <p>
+                                                            {avatar}Authored {formatDate(branch.changeDate)} ·{" "}
+                                                            {branch.changeHash?.substring(0, 8)}
+                                                        </p>
+                                                    )}
```

Comment:
**issue**: This is causing some duplicate elements to show up on the project **branches** and **prebuilds** pages.


| Branches| Prebuilds |
|-|-|
| <img width="1440" alt="Screenshot 2023-01-24 at 12 08 00 AM (2)" src="https://user-images.githubusercontent.com/120486/214160837-82cc2fef-56a0-4116-ae12-fd92bde9fda8.png"> | <img width="1440" alt="Screenshot 2023-01-24 at 12 07 56 AM (2)" src="https://user-images.githubusercontent.com/120486/214160842-26bf0fae-6757-4bdd-b917-5431c2082087.png"> |

Reply:
Looks good—resolving.